docs/getting-started-guides/digitalocean.md

```diff
@@ -38,15 +37,15 @@ As a minimum you will need to set these environment variables -
 
 ```
 # v2 API token
-DIGITALOCEAN_API_TOKEN=
+TF_VAR_do_token=
 ```
 
 To generate a v2 API token see [https://www.digitalocean.com/community/tutorials/how-to-use-the-digitalocea n-api-v2](https://www.digitalocean.com/community/tutorials/how-to-use-the-digitalocean-api-v2)
 
 ```
 # v1 API credentials as needed by ansible dynamic inventory.
-DO_CLIENT_ID=
-DO_API_KEY=
+APOLLO_do_client_id=
```

Comment:
you need to rebase since this got in https://github.com/Capgemini/Apollo/commit/9fcf6ee630f611e7d36ed65c939a6adb1e195f56